src/coreclr/src/vm/interoplibinterface.cpp

```diff
@@ -1198,6 +1247,78 @@ void ComWrappersNative::MarkExternalComObjectContextCollected(_In_ void* context
     }
 }
 
+void ComWrappersNative::MarkWrapperAsComActivated(_In_ IUnknown* wrapper)
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        MODE_ANY;
+        PRECONDITION(wrapper != NULL);
+    }
+    CONTRACTL_END;
+
+    InteropLib::Com::MarkComActivated(wrapper);
```

Comment:
This has a return value. Please comment as to why it is being ignored and/or at the least case indicate the return is explicitly being ignored.

`(void)InteropLib::Com::MarkComActivated(wrapper);`